openff/interchange/_tests/unit_tests/components/test_packmol.py

```diff
@@ -388,24 +388,6 @@ def test_packmol_add_negative_solvent_mass(self):
                 target_density=Quantity(1e-6, "kilogram/meter**3"),
             )
 
-    @pytest.mark.skipif(
-        has_package("mdtraj") or has_package("openmm"),
-        reason="Test requires that MDTraj **and** OpenMM are not installed",
-    )
-    def test_noninteger_serial_error(self):
-        """See issue #794."""
-        with pytest.raises(
-            PACKMOLRuntimeError,
-            match="could not be parsed by RDKit",
-        ):
-            pack_box(
-                molecules=[Molecule.from_smiles("CCO")],
-                number_of_copies=[11112],
-                box_shape=UNIT_CUBE,
-                tolerance=1.0 * unit.angstrom,
-                target_density=0.1 * unit.grams / unit.milliliters,
-            )
-
```

Comment:
(not blocking) This may have slipped in from another changeset - I'm not sure what removing this test has to do with changing config files/how OpenMM is handled in the test matrix. Not blocking, just wanted to make sure you saw this was here.

Reply:
Nice find, not sure how that snuck in there. Some tests are duplicated but this isn't such a case